### ISSUE TITLE
[no-implied-eval,no-throw-literal,no-empty-function] Documentation missing information that eslint no-implied-eval should be turned off.

### ISSUE THREAD

**Issue by radarsu** [typescript-eslint/typescript-eslint#2555]
- [ ] I have tried restarting my IDE and the issue persists.
- [ ] I have updated to the latest version of the packages.
- [x] I have [read the FAQ](https://github.com/typescript-eslint/typescript-eslint/blob/master/docs/getting-started/linting/FAQ.md) and my problem is not listed.

**Repro**

Go to:
https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-implied-eval.md
https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-throw-literal.md
https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-empty-function.md
Same rules exist for eslint:
https://eslint.org/docs/rules/no-implied-eval
https://eslint.org/docs/rules/no-throw-literal
https://eslint.org/docs/rules/no-empty-function

**Expected Result**

There should be snippet showing that eslint equivalent should be disabled when using @typescript-eslint/rule (it is done this way in all other cases) i.e.:
```
{
  // note you must disable the base rule as it can report incorrect errors
  "comma-spacing": "off",
  "@typescript-eslint/comma-spacing": ["error"]
}
```

**Actual Result**

There is no information if eslint equivalent should be turned off when using plugin.

**Comment by bradzacher** [comment]
This is all open source! Happy to accept a PR!

**Comment by radarsu** [comment]
PR addressing this issue created at https://github.com/typescript-eslint/typescript-eslint/pull/2578
